low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -71,8 +71,8 @@ LOGGINGMSGHANDLER:
 
 NODEDATAMSGHANDLER:
    transmit_interval: 10
-   high_cpu_usage_wait_threshold: 30
-   high_memory_usage_wait_threshold: 30
+   high_cpu_usage_wait_threshold: 10
+   high_memory_usage_wait_threshold: 20
```

Comment:
@gargpalak048, better way is to keep tests unaffected by actual sspl config, 
setting test values before test starts and resetting them at the end is better I think.

Can add the following in `setup/sspl_test.py` if everyone agrees 
```
            cpu_usage_alert_wait = Conf.get(SSPL_CONFIG_INDEX,
                "NODEDATAMSGHANDLER>high_cpu_usage_wait_threshold")

            memory_usage_alert_wait = Conf.get(SSPL_CONFIG_INDEX,
                "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold")

            cpu_usage_alert_wait_new = memory_usage_alert_wait_new = 0

            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_cpu_usage_wait_threshold",
                     cpu_usage_alert_wait_new)

            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold",
                     memory_usage_alert_wait_new)

            Conf.save(SSPL_CONFIG_INDEX)

#### ### run_qa_test.sh call   ######################            

            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_cpu_usage_wait_threshold",
                     cpu_usage_alert_wait)
            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold",
                     memory_usage_alert_wait)
```

Reply:
@sumedhak27, I believe 30s is a high time period to wait for sensors to raise a alert
Also, why to set the wait value as 0 while running test-cases?
shouldn't we test the sensors with actual config? 
> cpu_usage_alert_wait_new = memory_usage_alert_wait_new = 0

@jjxsg @cdeshmukh could you also share your opinions on this

Reply:
>better way is to keep tests unaffected by actual sspl config, setting test values before test starts and resetting them at the end is better I think.

@sumedhak27 is right, this is what i meant in scrum today. and not to change original sspl config itself.

Just before sanity tests, set these 
> high_cpu_usage_wait_threshold: 10
 high_memory_usage_wait_threshold: 20

restart sspl, run tests. revert config, restart sspl. This is what we do for current tests too